libs/net/models/Areas/Editor/Models/Report/ReportModel.cs

```diff
@@ -80,7 +80,7 @@ public Entities.Report ToEntity(JsonSerializerOptions options)
         entity.Settings = JsonDocument.Parse(JsonSerializer.Serialize(this.Settings, options));
         entity.Sections.ForEach(s =>
         {
-            var section = this.Sections.FirstOrDefault(us => us.Id == s.Id || us.Name == s.Name) ?? throw new InvalidOperationException("Unable to find matching section");
+            var section = this.Sections.FirstOrDefault(us => us.Name == s.Name) ?? throw new InvalidOperationException("Unable to find matching section");
```

Comment:
Fixed bug where saving multiple new sections would use the first  sections settings for all of the below.